### PR TITLE
Add Afaan Oromoo translations for 500 and 404 pages

### LINKED ISSUE #3287: Add translations for error pages for 4 live radio v1 services
**Is your feature request related to a problem? Please describe.**
We need to add the cookie / privacy banner translations for the 4 live radio v1 services

- Afaan Oromoo
- Amharic 
- Indonesia
- Korean 

**Describe the solution you'd like**
Add the translations to the service configs. Translations available on spreadsheet (pinned in ws-media slack channel or WSMEDIA-87)

**Testing notes**
[Tester to complete]

Dev insight: Unlikely to need a tester due to simplicity

**Additional context**
WSMEDIA-87

### PR DESCRIPTION
**Pull request by jroebu14**
Resolves https://github.com/bbc/simorgh/issues/3287

**Overall change:** Adds Afaan Oromoo translations for 500 and 404 pages.

**Code changes:**

- Added 500 and 404 translations to `src/app/lib/config/services/afaanoromoo.js`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

**Testing notes**

To observe the 404 error page
- go to http://localhost.bbc.com:7080/afaanoromoo/bbc_afaanoromoo_radio/blah

To observe the 500 error page
- add `status = 500` here https://github.com/bbc/simorgh/blob/404Handling/src/app/routes/getInitialData/utils/fetchData/index.js#L16
- go to http://localhost.bbc.com:7080/afaanoromoo

Translations can be verified here https://docs.google.com/spreadsheets/d/1IE6dvHfOJhYTIhC0kg8aRGnaj3fEg1NH9jQwo5NuUDQ/edit#gid=0

### CODE REVIEW

**Review by pjlee11 — APPROVED** [review]

**Review by dr3 — COMMENTED** [review]
Dont speak this language but LGTM assuming translations are tested before merge

**Review by Bopchy — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**
